examples/point-of-sale/src/server/core/cache.ts

```diff
@@ -1,5 +1,3 @@
 import cacheManager from 'cache-manager';
-// @ts-ignore (TS7016) There is no type definition for this at DefinitelyTyped.
-import MemoryStore from 'cache-manager/lib/stores/memory';
 
-export const cache = cacheManager.caching({ store: MemoryStore, max: 1000, ttl: 120 /*seconds*/ });
+export const cache = cacheManager.caching('memory', { max: 1000, ttl: 120 /*seconds*/ });
```

Comment:
Is this change the only reason to update the package?

Reply:
Nope. 
This is a collateral.
Main reasons are to use solana mobile adapter and get rid of all the warning when compiling